src/main/java/picard/arrays/illumina/CreateExtendedIlluminaManifest.java

```diff
@@ -0,0 +1,701 @@
+
+    @Argument(shortName = StandardOptionDefinitions.OUTPUT_SHORT_NAME, doc = "The name of the extended manifest to be written.")
+    public File OUTPUT;
+
+    @Argument(shortName = "BAF", doc = "The name of the the 'bad assays file'. This is a subset version of the extended manifest, " +
+            "containing only unmappable assays", optional = true)
+    public File BAD_ASSAYS_FILE;
+
+    @Argument(shortName = "RF", doc = "The name of the the report file")
+    public File REPORT_FILE;
+
+    @Argument(shortName = "FD", doc = "Flag duplicates in the extended manifest.  " +
+            "If this is set and there are multiple passing assays at the same site (same locus and alleles) " +
+            "then all but one will be marked with the 'DUPE' flag in the extended manifest. " +
+            "The one that is not marked as 'DUPE' will be the one with the highest Gentrain score as read from the cluster file.", optional = true)
+    public Boolean FLAG_DUPLICATES = true;
+
+    @Argument(shortName = "CF", doc = "The Standard (Hapmap-trained) cluster file (.egt) from Illumina. " +
+            "If there are duplicate assays at a site, this is used to decide which is the 'best' (non-filtered in generated VCFs) " +
+            "by choosing the assay with the best GenTrain scores)", optional = true)
+    public File CLUSTER_FILE;
+
+    @Argument(shortName = "DBSNP", doc = "Reference dbSNP file in VCF format.", optional = true)
+    public File DBSNP_FILE;
+
+    @Argument(shortName = "TB", doc = "The target build.  This specifies the reference for which the extended manifest will be generated. " +
+            "Currently this tool only supports Build 37 (Genome Reference Consortium Human Build 37 (GRCh37)). " +
+            "If entries are found in the Illumina manifest that are on this build they will be used with the coordinate specified in the manifest, " +
+            "If there are entries found on other builds, they will be marked as failed in the extended manifest UNLESS the " +
+            "build and liftover information (SUPPORTED_BUILD, SUPPORTED_REFERENCE_FILE, and SUPPORTED_CHAIN_FILE) is supplied.")
+    public String TARGET_BUILD = "37";
+
+    @Argument(shortName = "SB", doc = "A supported build. The order of the input must match the order for SUPPORTED_REFERENCE_FILE and SUPPORTED_CHAIN_FILE. " +
+            "This is the name of the build as specified in the 'GenomeBuild' column of the Illumina manifest file.",
+            optional = true)
+    public List<String> SUPPORTED_BUILD;
+
+    @Argument(shortName = "SR", doc = "A reference file for the provided SUPPORTED_BUILD. " +
+            "This is the reference file that corresponds to the 'SUPPORTED_BUILD' as specified above.",
+            optional = true)
+    public List<File> SUPPORTED_REFERENCE_FILE;
+
+    @Argument(shortName = "SC", doc = "A chain file that maps from SUPPORTED_BUILD -> TARGET_BUILD. Must provide a corresponding supported reference file.",
```

Comment:
Yeah, I need to add a whole other usage example for this.  I fear that this will cause some confusion among users.  Fortunately the need to liftover doesn't happen that often.

Reply:
Added usage for liftover case.